docs/source/release_notes/release-notes.rst

```diff
@@ -50,8 +50,25 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |new| |improved| |metrics| |logs|
+         - Sweep metrics were reworked. Sweep now exposes metrics indicating the total number of cells examined, cells deleted, time spent sweeping, and time elapsed since sweep started on the current table that are updated after each iteration of sweep and separate metrics that are updated after each table is fully swept.
+           Additionally, sweep now exposes metrics tagged with table names that expose the total number of cells examined, cells deleted, time spent sweeping per iteration for each table separately.
+           Logs will also include the new timing information.
+           Sweep now exposes the following metrics with the common prefix ``com.palantir.atlasdb.sweep.metrics.SweepMetric.``:
+
+              - ``cellTimestampPairsExamined.meter.perIteration``
+              - ``cellTimestampPairsExamined.histogram.perTable``
+              - ``cellTimestampPairsExamined.histogram.perIteration`` (tagged)
+              - ``staleValuesDeleted.meter.perIteration``
+              - ``staleValuesDeleted.histogram.perTable``
+              - ``staleValuesDeleted.histogram.perIteration`` (tagged)
+              - ``sweepTimeSweeping.meter.perIteration``
+              - ``sweepTimeSweeping.histogram.perTable``
+              - ``sweepTimeSweeping.histogram.perIteration`` (tagged)
+              - ``sweepTimeElapsedSinceStart.currentValue.perIteration``
+              - ``sweepTimeElapsedSinceStart.histogram.perTable``
```

Comment:
My intuition is that any of the `histogram.perTable` metrics are going to be hard to reason about.  Happy to include for now and then look at the graphs, but if they aren't answering a specific question then we should remove them.